packages/compiler/core/scanner.ts

```diff
@@ -869,8 +873,23 @@ export function createScanner(
   }
 
   function getIdentifierTokenValue(): string {
-    const text = getTokenText();
-    return tokenFlags & TokenFlags.NonAscii ? text.normalize("NFC") : text;
+    const start = tokenFlags & TokenFlags.Backticked ? tokenPosition + 1 : tokenPosition;
+    const end =
+      tokenFlags & TokenFlags.Backticked && !(tokenFlags & TokenFlags.Unterminated)
+        ? position - 1
+        : position;
+
+    if (tokenFlags & TokenFlags.Escaped) {
+      return unescapeBacktickedString(start, end);
```

Comment:
Assuming we do keep escaping and non-ascii characters (pending discussing on Teams), we still need to normalize if we unescape, so this should set the text to either getTokenText() or unescapedBacktickedString() and continue to the normalization.

Reply:
fixed